router/server.go

```diff
@@ -102,6 +103,29 @@ func main() {
 	apiPort := flag.String("api-port", "", "api listen port")
 	flag.Parse()
 
+	if ports := os.Getenv("HTTP_PORTS"); ports != "" {
```

Comment:
I think the env var should be something like `ADDITIONAL_HTTP_PORTS` which adds ports to those specified by the flags (i.e. 80 / 443) which avoids users shooting themselves in the foot and ending up with the cluster not listening on 80 / 443, WDYT?